.github/workflows/test.yml

```diff
@@ -61,14 +66,20 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled }}
 
-      - name: Test with pytest (Linux)
+      - name: Run Integration Tests with pytest (Linux)
         if: runner.os == 'Linux'
+        env:
+          BORG_VERSION: ${{ matrix.borg-version }}
         run: |
           xvfb-run --server-args="-screen 0 1024x768x24+32" \
                    -a dbus-run-session -- make test
-      - name: Test with pytest (macOS)
+
+      - name: Run Integration Tests with pytest (macOS)
         if: runner.os == 'macOS'
-        run: make test
+        env:
+          BORG_VERSION: ${{ matrix.borg-version }}
+          PKG_CONFIG_PATH: /usr/local/opt/openssl@3/lib/pkgconfig
+        run: echo $PKG_CONFIG_PATH && make test
```

Comment:
`make test` runs unit test for all borgbackup version which is unnecessary. Should I add a condition to run `make test` only for borgbackup version 1.1.18 and `make test-integration` for the rest, if that makes sense?

Reply:
Can you put the unit tests in a separate run which doesn't install borg at all (or do we need a borg installation for the unit tests to work)?

Reply:
@real-yfprojects We do need some installation of borg so Vorta doesn't complain at startup and maybe during other functions.

Reply:
I would still have a separate run for the unittests so that it shows up separately in the PR checks list.

Reply:
I've added a seperate job for unit tests instead of trying to accomodate in main job to reduce complexity. I tried to use include to create combination which includes unit tests for just one version but that didn't work properly.